SmellDetector/Enumerations/OperationEnum.cs

```diff
@@ -0,0 +1,8 @@
+﻿public enum OperationEnum
```

Comment:
Enums should also be a part of the namespace. This file should be moved alongside the Criteria.